content/styles/04-templates/home.css

```diff
@@ -1,5 +1,11 @@
-.home {
+/* .home {
```

Comment:
Why commenting instead of removing?

Reply:
OMG... sunday work 😛 adressed with dc24ef155462d8b4998f5703d38f90e53c206b97